storybookjs/storybook
# Angular: Disable prod mode for debugging components

Issue:

## What I did
- I disabled prod mode to enable dev tools.
Now its possible to debug components with ng.probe:
![image](https://user-images.githubusercontent.com/1221055/54750275-68cfa000-4bd7-11e9-953f-d25a02b6e55b.png)

## How to test

- Is this testable with Jest or Chromatic screenshots?
  - No
- Does this need a new example in the kitchen sink apps?
  - No
- Does this need an update to the documentation?
  - No
If your answer is yes to any of these, please make sure to include it in your PR.
